osu.Game/Skinning/ISerialisableDrawable.cs

```diff
@@ -27,6 +27,14 @@ public interface ISerialisableDrawable : IDrawable
         /// </summary>
         bool IsEditable => true;
 
+        /// <summary>
+        /// Whether this component supports the "closest" anchor.
+        /// </summary>
+        /// <remarks>
+        /// This is disabled by some components that shift position automatically.
```

Comment:
What's an example of this? I don't quite understand.

Reply:
Disabled by `LegacyManiaComboCounter` because its position is shifted when the user flips the scrolling direction. I don't recall why though.

Reply:
This seems like an odd choice. So the user can still choose an anchor and it will just flip? I'd say that closest should still be allowed and the flip is only applied if the counter is in a certain region of the screen (or have a certain anchor applied).

Reply:
I've removed this for now as I can't really understand why it is required. It would make more sense to completely disable anchors (or a certain subset) but honestly I'd just leave this up to the user, and adjust the automatic flipping logic to only work in the cases where it makes sense.